README.Rmd

```diff
@@ -13,12 +13,30 @@ knitr::opts_chunk$set(
 )
 ```
 
-# RSimplerEDA
+# RSimplerEDA - A Package to Simplify Some Common Plotting for Exploratory Data Analysis (EDA)
 
 <!-- badges: start -->
 <!-- badges: end -->
 
-The goal of RSimplerEDA is to ...
+
+## Overview
+
+Exploratory Data analysis (EDA) is an important step in any data analysis. 
+However, carrying out EDA with the Altair package requires a lot of coding effort. 
```

Comment:
However, carrying out EDA with the ggplot2 package requires a lot of coding effort.